### DISCUSSION TITLE
Which target for ESP32 D1 Mini NodeMCU?

### DISCUSSION THREAD

**Discussion by MarkuBu** [esp-rs/esp-template#170] [Q&A] [answered]
I want to use ESP32 D1 Mini NodeMCU for some projects, but I don't know which target if I should use for the esp-template. The datasheet lists ESP32-D0WD-V3 or ESP32-D0WDR2-V3 for the ESP32 Wroom modules.

Are they supported by the template?

**Comment by SergioGasquez** [comment, marked as answer]
> Are they supported by the template?

Yes, both modules are ESP32, and they are supported.
